src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -24,19 +24,31 @@
 		role="dialog"
 		:aria-label="t('spreed', 'Conversation settings')"
 		:open.sync="showSettings"
-		:show-navigation="false">
+		:show-navigation="true">
+		<!-- Guest access -->
 		<AppSettingsSection
 			:title="t('spreed', 'Guests access')"
 			class="app-settings-section">
 			<LinkShareSettings ref="linkShareSettings" />
 		</AppSettingsSection>
+		<!-- Notifications settings -->
+		<AppSettingsSection
+			:title="t('spreed', 'Notifications')"
```

Comment:
```suggestion
			:title="t('spreed', 'Chat notifications')"
```
To not break existing translations